CGMBLEKit Example/CommandQueue.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  CommandQueue.swift
+//  CGMBLEKit Example
+//
+//  Created by Paul Dickens on 25/03/2018.
+//  Copyright © 2018 LoopKit Authors. All rights reserved.
+//
+
+import Foundation
+import xDripG5
+
+
+class CommandQueue {
+    private var list = Array<Command>()
+    private var lock = os_unfair_lock()
+
+    var isEmpty: Bool {
+        return list.isEmpty
```

Comment:
This doesn't have any callers, so it should either be removed or moved inside the lock.